conditional-middleware/index.js

```diff
@@ -21,15 +19,22 @@ function ignoreAssets(mw) {
   }
 }
 
+function *responseTime(next) {
+  var start = new Date;
+  yield next;
+  var ms = new Date - start;
+  this.set('X-Response-Time', ms + 'ms');
+}
+
```

Comment:
Totally, I was just following the use of `responseTime` in the other examples. https://github.com/koajs/examples/blob/4bb4c5f5f15fdc9a2cf5fe9efb5e9cfff0a19624/compose/app.js#L22 Didn't think it was worth it to add another dep.